homeworkW13D5/src/homeworkW13D5/ORM/Model.java

```diff
@@ -0,0 +1,21 @@
+package homeworkW13D5.ORM;
+public abstract class Model {
+
+	@SuppressWarnings("unused")
+	private String tableName;
+
+	public Model(String tableName) {
+		this.tableName = tableName;
+	}
+
+	public String findByPk(Integer id, String tableName) {
```

Comment:
Why require table name. It is already known and stored in 'tableName' private member.